app/controllers/application_controller.rb

```diff
@@ -1,21 +1,24 @@
+# frozen_string_literal: true
+
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
   before_action :configure_permitted_parameters, if: :devise_controller?
   rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
   rescue_from ActionController::RoutingError, with: :route_not_found
 
-	protected
+  protected
+
   def configure_permitted_parameters
-  	added_attrs = [:username, :email, :password, :password_confirmation, :remember_me]
+    added_attrs = %i[username email password password_confirmation remember_me]
```

Comment:
permitted_attributes would make more sense i guess.

added gives out the wrong impression.